src/screens/Search/components/SearchList/index.tsx

```diff
@@ -1,14 +1,65 @@
-import React from 'react';
-import { Index } from 'react-instantsearch-native';
+import React, { useEffect, useState } from 'react';
+import { Index, connectRefinementList } from 'react-instantsearch-native';
 import { List } from './styles';
 
 interface SearchListProps {
   index: string
   type: string
+  filters?: {
+    refinementList?: {
+      [key: string]: any
+    }
+  }
 }
 
-export default ({ index, type }: SearchListProps): JSX.Element => (
-  <Index indexName={index}>
-    <List type={type} />
-  </Index>
-);
+interface FilterStates {
+  [key: string]: any
+}
+
+const Refinement = connectRefinementList(() => null);
```

Comment:
Hey @davioliveiira! Could you show how the filters on this screen are working? 👀

Just a note: these filters are only for the technology index. When we need to list the services, for example, this component would no longer be useful because this refinement list is static on your component now. I suggest you take a look at the web package solution for this problem 🤓

Reply:
@mateus4k estou subindo um fix para organizar melhor isso, tem algumas coisas que no native são diferentes, pois a lib de algolia não tem suporte a tudo que temos na web 😢